README.md

```diff
@@ -49,26 +54,26 @@ export PKG_CONFIG_PATH=/opt/homebrew/Cellar/openssl@1.1/1.1.1l_1/lib/pkgconfig/
 
 #### Ubuntu
 
-```
+```bash
 sudo apt-get install libsrtp2-dev libavcodec-dev libavformat-dev libavutil-dev
 ```
 
 ### Launching of the application
 
 First, install all project dependencies:
 
-```
+```bash
 mix deps.get
 npm ci --prefix=assets
 ```
 
 To run, type:
 
-```
+```bash
 EXTERNAL_IP=<IPv4> mix phx.server
 ```
 
-`EXTERNAL_IP` should be set to the local IP address of the computer this is running on.
+`EXTERNAL_IP` should be set to the local IP address of the computer this is running on. `EXTERNAL_IP` is optional.
```

Comment:
Without it you won't be able to connect from outside of `localhost`. I'd rather rephrase this like:
```suggestion
`EXTERNAL_IP` should be set to the local IP address of the computer this is running on.
It is required unless you only connect via `localhost`.
```